lib/bobby.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const AWS = require('aws-sdk');
+const CONFIGS = {
+    pipelines: require('../config/pipeline') // eslint-disable-line global-require
+};
+const dydb = new AWS.DynamoDB({
+    region: 'us-west-2'
```

Comment:
It's now configurable.

This is to give the function a default region. The function caller can override it on a per-call basis.